examples/function_memory_game/src/components/chessboard_card.rs

```diff
@@ -42,7 +42,7 @@ pub fn ChessboardCard(props: &Props) -> Html {
 
     html! {
       <div class="chess-board-card-container">
-          <div class={classes!("card", flipped.then(|| "flipped"))} {onclick}>
+          <div class={classes!("card", flipped.then(|| "flipped"))} onclick={Callback::from(onclick)}>
```

Comment:
I would prefer you created the Callback higher and keep html! clean of rust code mostly

Reply:
Oh that change could be reverted now. `Callback::from` is no longer needed here